sdk/attestation/azure-security-attestation/src/test/java/com/azure/security/attestation/AttestationTest.java

```diff
@@ -258,7 +258,11 @@ void testAttestSgxEnclaveAsync(HttpClient httpClient, String clientUri) {
                         }
                         logger.info(String.format("Issuer of signing certificate is: %s", signer.getCertificates().get(0).getIssuerDN().getName()));
                     }
-                }));
+                })
+                // Only validate time based properties when not in PLAYBACK mode. PLAYBACK mode has these values
+                // hard-coded into the session record.
+                .setValidateNotBefore(getTestMode() != TestMode.PLAYBACK)
```

Comment:
I'm not sure that you need to disable NotBefore checking, but it won't hurt to have it.